service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/RespondentSolicitorContentProvider.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
+import uk.gov.hmcts.reform.fpl.model.notify.submittedcase.RespondentSolicitorTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.SharedNotifyContentProvider;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class RespondentSolicitorContentProvider extends SharedNotifyContentProvider {
+
+    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookup;
+
+    public RespondentSolicitorTemplate buildRespondentSolicitorSubmissionNotification(
+        CaseData caseData, RespondentSolicitor representative) {
+
+        return RespondentSolicitorTemplate.builder()
+            .representativeName(representative.getFirstName() + " " + representative.getLastName())
+            .localAuthority(localAuthorityNameLookup.getLocalAuthorityName(caseData.getCaseLocalAuthority()))
```

Comment:
we can use 
`            .localAuthority(caseData.getCaseLocalAuthorityName())`
as this is set in case initiation which will call the name lookup

Reply:
discussed, this is fine as is 👍